### ISSUE TITLE
Who can I specify MS-DOS as File system?

### ISSUE THREAD

**Issue by shy-neon** [stoqn4opm/DiskUtil#1]
the only suggested filesystem by Xcode is .apfs, any hint?

**Comment by stoqn4opm** [comment]
i am sorry, what is the ask here?

Did you had a look at the file `Sources/DiskUtil/FileSystem.swift`? Uncomment the cases inside the file system `enum` there and give it a try 👍

**Comment by shy-neon** [comment]
yes, I actually found out that. Why are them commented on the first place?

**Comment by stoqn4opm** [comment]
i don' remember it was really long ago, fork the repo uncomment those and use it. It should be fine, as its passing it straight to Apple's classes
